### PR TITLE
call serverside to clientside

### PR DESCRIPTION
**Pull request by CloggedToaster**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
-
+import React from "react"
```
Comment (noahallen) [inline client/src/App.js]: This should probably include useState and useEffect

File: client/src/App.js [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
 import './App.css';
 import Routing from "./routes/index";
 
-
 function App() {
+
+  const [data, setData] = React.useState(null);
```
Comment (noahallen) [inline client/src/App.js]: shouldn't be React.useState just useState

**Review by noahallen — CHANGES_REQUESTED** [review]
Small comments

**Review by pzjiang — APPROVED** [review]
nice

File: client/src/App.js (hunk 1 above)
Comment (CloggedToaster) [reply client/src/App.js]: Added useState and useEffect to import

File: client/src/App.js (hunk 2 above)
Comment (CloggedToaster) [reply client/src/App.js]: removed React.useState and React.useEffect

**Review by noahallen — APPROVED** [review]

**Pull request merged**